Cart-management/cart-management.css

```diff
@@ -0,0 +1,264 @@
+@import url(/Utility/utility.css);
+
+.icon {
+    color: var(--color-light-gray);
+    font-size: 2rem;
+}
+
+.inner-icon {
+    color: var(--color-light-gray);
+    font-size: 1.3rem;
+    padding: 1rem;
+}
+
+.input-box button {
+    padding: 1.2rem;
+    outline: none;
+}
+
+button {
+    border-radius: 0;
```

Comment:
The border radius is by default zero

Reply:
yes I agree but the component is taken from the component library and there I have initialized some border-radius so here it is cancelling the effect.